src/Components/ResourceList/Card/ResourceCardMeta.tsx

```diff
@@ -0,0 +1,72 @@
+// Libraries
+import React, {forwardRef} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {FlexBox, FlexBoxProps, FlexBoxRef} from '../../FlexBox'
+
+// Types
+import {Omit, FlexDirection, AlignItems} from '../../../Types'
+
+// Styles
+import './ResourceCardMeta.scss'
+
+export interface ResourceCardMetaProps
+  extends Omit<
+    FlexBoxProps,
+    'stretchToFitWidth' | 'stretchToFitHeight' | 'margin'
+  > {}
+
+export type ResourceCardMetaRef = FlexBoxRef
+
+export const ResourceCardMeta = forwardRef<
+  ResourceCardMetaRef,
+  ResourceCardMetaProps
+>(
+  (
+    {
+      id,
+      style,
+      testID = 'resource-list--meta',
+      children,
+      className,
+      direction = FlexDirection.Row,
+      alignItems = AlignItems.Center,
+      justifyContent,
+    },
+    ref
+  ) => {
+    const resourceCardMetaClass = classnames('cf-resource-meta', {
+      [`${className}`]: className,
+    })
+
+    let wrappedChildren = children
```

Comment:
```suggestion
    let wrappedChildren
```

a little more explicit and less confusing. we only want to use `children` if there are children, which is checked on the line below. this way is effectively the same thing, but doesn't require guessing what the value of `children` is.